docs/api-docs/channels/channels-toolkit-reference.md

```diff
@@ -26,36 +26,36 @@ The Channel Toolkit is a set of APIs, UI components, patterns, and app framework
 
 | Endpoint | Description |
 |-|-|
-|[Channels](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)||
-|[Listings](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)||
-|[Orders](https://developer.bigcommerce.com/api-reference/store-management/orders)||
-|[Products](https://developer.bigcommerce.com/api-reference/store-management/catalog)||
+|[Channels](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)|Create and manage product listings for multiple storefronts and sales channels|
```

Comment:
@aglensmith Should these descriptions be the same?

Reply:
Yeah, they're two parts of a whole.